doc/nrf/releases_and_maturity/migration_guides.rst

```diff
@@ -9,10 +9,18 @@ Migration guides are also provided for major functionality updates.
 .. note::
     |migration_contact_devzone|
 
-.. toctree::
-   :maxdepth: 1
-   :glob:
-   :reversed:
-   :caption: Subpages:
+* `Migrating from multi-image builds to sysbuild`_
+* `Migrating to the current hardware model`_
+* `Migration guide for nRF Connect SDK v2.9.0`_
+* `Migration guide for nRF Connect SDK v2.8.0`_
+* `Migration guide for nRF Connect SDK v2.7.0`_
+* `Migration guide for nRF Connect SDK v2.6.0`_
+* `Migration guide for nRF Connect SDK v2.5.0`_
+* `Migration guide for nRF Connect SDK v2.0.0`_
 
-   migration/*
+For nRF54H20 customer sampling participants:
```

Comment:
Hi @peknis, this categorization is confusing, nRF54H20 users should probably use all the pages listed in this page. However, the ones listed below are exclusively for customer sampling releases. So we should rephrase this as something like " For nRF54H20-exclusive customer sampling releases." These are just for nRF54, but the previous ones are also for nRF54 (except 2.9.0)